modules/nw-networkpolicy-create.adoc

```diff
@@ -1,57 +1,84 @@
 // Module included in the following assemblies:
 //
 // * networking/network_policy/creating-network-policy.adoc
-// * networking/configuring-networkpolicy.adoc
 // * post_installation_configuration/network-configuration.adoc
 
 [id="nw-networkpolicy-create_{context}"]
 
 = Creating a network policy
 
-To define granular rules describing ingress network traffic allowed for projects
-in your cluster, you can create a network policy.
+To define granular rules describing ingress network traffic allowed for namespaces in your cluster, you can create a network policy.
```

Comment:
Yea if we are mentioning ingress/ and egress, It would also make sense to include an egress policy example too.

Reply:
@anuragthehatter, the work for that is in this PR https://github.com/openshift/openshift-docs/pull/33350. My goal is to avoid too many examples in the procedures, but to include detailed examples in the conceptual documentation.